src/quickgui/plugin/qmldir

```diff
@@ -16,5 +16,6 @@ plugin qgis_quick_plugin
 MapCanvas 1.0 qgsquickmapcanvas.qml
 FeatureForm 1.0 qgsquickfeatureform.qml
 PositionMarker 1.0 qgsquickpositionmarker.qml
+PhotoCapture 1.0 qgsquickphotocontainer.qml
```

Comment:
Could you please update the .qml file name so that it matches the QML class name?